<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frm_Main</class>
 <widget class="QWidget" name="frm_Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Electronic Order System</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>resources/user.png</normaloff>resources/user.png</iconset>
  </property>
  <widget class="QLabel" name="Label_Menu">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>441</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:xx-large; font-weight:600; color:#008000;&quot;&gt;Welcome to our restaurant!&lt;/span&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:large; font-weight:50; color:#ff0000;&quot;&gt;    please select:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>361</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="Button_View">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string> View the menu list</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>resources/menu.png</normaloff>resources/menu.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Button_ViewMine">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string> View your order    </string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>resources/menu.png</normaloff>resources/menu.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Button_Exit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string> Pay and exit          </string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>resources/cancel.png</normaloff>resources/cancel.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
